docs/intro.rst

```diff
@@ -44,15 +44,9 @@ Then you can build all the services::
 
    $ docker-compose -f docker/docker-compose.dev.yml build
 
-You need MusicBrainz database containing all the MusicBrainz music metadata for setting up your application. Data dumps provided from https://musicbrainz.org can be downloaded from https://musicbrainz.org/doc/MusicBrainz_Database/Download. ``mbdump.tar.bz2`` is the core MusicBrainz database including the tables for artist, release groups etc. ``mbdump-derived.tar.bz2`` contains annotations, user tags and search indexes. The core and the derived database covers all the data required for a CritiqueBrainz server.
+MusicBrainz database containing all the MusicBrainz music metadata is needed for setting up your application. The ``mbdump.tar.bz2`` is the core MusicBrainz database including the tables for artist, release_group etc. ``mbdump-derived.tar.bz2`` contains annotations, user tags and search indexes. The core and the derived database covers all the data required for a CritiqueBrainz server.
```

Comment:
> all the MusicBrainz music metadata

I think "music" part can be removed since it's not only music.

> The ``mbdump.tar.bz2`` is the core MusicBrainz database

It's not really a database. Can call it a database dump or an archive, otherwise it can get confusing.

> for setting up your application

_for setting up an instance of CritiqueBrainz_

> including the tables for artist, release_group etc.

_which includes the tables for artist, release group, and other entities_

> The core and the derived database covers

_These archives include